### PR TITLE
Small fixes, fix copy paste and remove 'm_' variable prefix

### PR DESCRIPTION
**Pull request by DanVanAtta**
This change is [Reviewable](https://reviewable.io/reviews/triplea-game/triplea/587)

### CODE REVIEW

File: src/games/strategy/util/SoftJEditorPane.java [hunk 1]
```diff
@@ -10,47 +10,47 @@
...
   }
 
   public synchronized JEditorPane getComponent() {
-    if (m_component == null) {
-      m_component = new SoftReference<JEditorPane>(createComponent());
+    if (component == null) {
+      component = new SoftReference<JEditorPane>(createComponent());
     }
-    JEditorPane component = m_component.get();
+    JEditorPane component = this.component.get();
```
Comment (ron-murhammer) [inline src/games/strategy/util/SoftJEditorPane.java]: Can we rename the local variable so we don't have a local and member variable with the same name?

File: src/games/strategy/util/SoftJEditorPane.java [hunk 2]
```diff
@@ -10,47 +10,47 @@
...
   public String getText() {
-    return m_text;
+    return text;
   }
 
   public void dispose() {
-    if (m_component != null) {
-      JEditorPane component = m_component.get();
+    if (component != null) {
+      JEditorPane component = this.component.get();
```
Comment (ron-murhammer) [inline src/games/strategy/util/SoftJEditorPane.java]: Can we rename the local variable so we don't have a local and member variable with the same name?

**Comment by ron-murhammer** [comment]
@DanVanAtta A few comments added.

File: src/games/strategy/util/SoftJEditorPane.java (hunk 1 above)
Comment (DanVanAtta) [inline src/games/strategy/util/SoftJEditorPane.java]: Renamed: c03d2d9
Also deprecated this class, noticing the usage of a soft reference. That usage makes it horribly complicated, and very unclear how much memory we are actually saving.

File: src/games/strategy/util/SoftJEditorPane.java (hunk 2 above)
Comment (DanVanAtta) [inline src/games/strategy/util/SoftJEditorPane.java]: renamed.

**Pull request merged**